Add functions to support .md files

suhhee1011 opened the pull request:
This pull request resolves #9  . I added a feature to support markdown. Please, let me know by comment if it needs some more features to add or codes need to be fixed.


**What I changed:**

**In generateHTMLStr function,**

1.  Added regular expression to convert markdown files to HTML files
    - Find bold, italic, anchor, h1, and h2 tags
2. Add conditions to distinguish if it is txt files or MD files
    - Convert bold, italic, anchor, h1, and h2 tags

**In strToFile function,**

1. Added to let users can convert MD files to HTML.
2. Both MD files and MD files with a folder can convert to HTML

mkim219 commented:
I have reviewed your PR. There are a few things need to be changed:

- If the ,md file in the folder with other text file, it should be also converted to HTML file
- There are unnecessary file you have committed. Please add those files in .gitignore and delete the files on your branch

suhhee1011 commented:
Thank you for your comment. Based on your comment, I fixed the following:
 - Create a dist folder to the main directory instead of inside the text folder. Applied both file type text and MD.
 - Added .gitignore and add bin and obj to remove the unnecessary files

mkim219 commented:
Thank you for finding new bug that I could not found. Thank you for your work!
